python/tvm/driver/tvmc/compiler.py

```diff
@@ -223,6 +223,7 @@ def compile_model(
     use_vm: bool = False,
     mod_name: Optional[str] = "default",
     workspace_pools: Optional[WorkspaceMemoryPools] = None,
+    instruments = None,
```

Comment:
nit: could we add the `Optional[Sequence[PassInstrument]]` type annotation

Reply:
> nit: could we add the `Optional[Sequence[PassInstrument]]` type annotation

done